packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process..st

```diff
@@ -0,0 +1,4 @@
+accessing
+process: anOSProcess
+
+	Process := anOSProcess
```

Comment:
```suggestion
accessing
process: anOSProcess

	process := anOSProcess
```
Don't use class variable names, use instance variables on class side instead (and start with small letter)